stdlib/random.ml

```diff
@@ -93,6 +93,22 @@ module State = struct
     then invalid_arg "Random.int"
     else intaux s bound
 
+  let rec int63aux s n =
+    let b1 = bits s in
+    let b2 = bits s lsl 30 in
+    let b3 = (bits s land 3) lsl 60 in
```

Comment:
```suggestion
    let b3 = (bits s lsr 28) lsl 60 in
```
It's better to use the high-order bits because (as far as I remember) they are slightly more random.